crates/spawner/src/spawner.rs

```diff
@@ -78,7 +78,7 @@ fn spawn(
                 let player = *player.expect("Active object without an associated was spawned.");
                 counter.player_mut(player).unwrap().update(active_type, 1);
 
-                if player == game_config.player() {
+                if player == game_config.player() || cfg!(debug_assertions){
```

Comment:
IMHO `debug_assertions` should produce the same functionality (just a bit slower & with more assertions). Its main purpose is to make bug / wrong assumption discovery easier.

Sorry for not making #249 clear. We should hide this behind a [feature](https://doc.rust-lang.org/cargo/reference/features.html) so one can `cargo run --features enemy`.